ocaml/typing/typetexp.ml

```diff
@@ -465,7 +465,17 @@ and transl_type_aux env policy mode styp =
         match args with
         | (l, arg_mode, arg) :: rest ->
           check_arg_type arg;
-          let arg_cty = transl_type env policy arg_mode arg in
+          let l, arg_cty =
+            match l, arg.ptyp_desc with
+            | (Labelled l | Position l), Ptyp_extension ({txt="src_pos"; _}, _payload) ->
+              (* CR src_pos: This won't work correctly in Untypeast *)
+              (* TODO vding question: I think the above is no longer true? *)
```

Comment:
Looking for confirmation on this. I think because we use a Predef and no manually constructed Longidents, we're off the hook for this problem?